pulp_python/app/tasks/sync.py

```diff
@@ -119,9 +119,8 @@ def _fetch_specified_metadata(remote, project_specifiers):
                     else:
                         for type, digest in package['digests'].items():
                             if digests.filter(type=type, digest=digest).exists():
-                                remote_units.append(parse_metadata(metadata['info'],
-                                                                   version,
-                                                                   package))
+                                remote_units.append(
+                                    parse_metadata(metadata['info'], version, package))
```

Comment:
People may have diverging opinions on having the ending paren on the same line vs. next line.  I kinda like it this way but I don't feel strongly, and the 2nd way is probably easier to make consistent.

```
remote_units.append(
   version, parse_metadata(metadata['info'], version, package))
```
vs.
```
remote_units.append(
   version, parse_metadata(metadata['info'], version, package)
)
```

Reply:
I have a slight preference for the 2nd way, but I don't feel too strongly either way

Reply:
Changed